.github/workflows/binary-compatibility-checks.yml

```diff
@@ -23,7 +23,7 @@ jobs:
         # The versions of scala specified here are only used as `+~...`
         # which ignores the PATCH portion of the version id. These Scala versions don't need
         # to be fully defined here then since Akka build handles which patch version will be used.
-        scalaVersion: [ "2.12", "2.13" ]
+        scalaVersion: [ "2.12", "2.13", "3.1" ]
```

Comment:
Does this mean that it will run all 9 combinations? Previously 6.
I think it's overkill to test all combinations.

Reply:
misplaced comment ^. It was about running tests with combinations of scala and jdk

Reply:
This is as before on JDK 11 only.